commit-message-hook.sh

```diff
@@ -28,3 +28,23 @@ fi
     echo "Branch name is not named properly, please see wiki for formating: https://github.com/space-concordia-robotics/robotics-prototype/wiki/Work-Flow"
     exit 1
 fi
+#Checks to see if branch name includes any upper case letters. If it does, it exits and tells user to check wiki
+if [[ $BRANCH_NAME =~ [\$A-Z] ]];then
+# If any upper case letters are in the branch name,
+  echo "Branch name is not named properly, please see wiki for formating: https://github.com/space-concordia-robotics/robotics-prototype/wiki/Work-Flow"
```

Comment:
This same string keeps being repeated again and again, so we should store it in a variable at the top of the script and just use the variable underneath instead.

Reply:
Stored the string under the variable "ERROR"